drivers/cadence/uart/cdns_console.S

```diff
@@ -0,0 +1,127 @@
+/*
```

Comment:
Does this uart have a part number or some other way to distinguish it? If so can this be reflected in the filenames? If this is only found in the Xilinx SoCs then it's better to keep it in the platform port.